.github/workflows/kythe.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Install Kythe, extract Kzips, and upload to GCP to enable semantic indexing on
+# https://cs.opensource.google/opentitan
+
+name: kythe
+
+on:
+  schedule:
+    # Run on the 47th minute every 6 hours.
+    - cron: '47 */6 * * *'
+
+jobs:
+  build_and_upload_kzips:
+    name: 'Build and upload kzips for Kythe/Codesearch'
+    runs-on: ubuntu-latest
```

Comment:
I don't think that's necessary here. For the main CI we are testing OpenTitan itself functions correctly with the documentation minimum environment (which is based on Ubuntu 18.04). This is just publishing OT code to another index, it's not something that's really part of supported OT functionality so it's reasonable to just do whatever needed for it to work and we can ignore the standard OT environment setup.

Reply:
One side benefit of using the latest release is that it might give us a heads up on distro changes that would break our build. The downside is a lack of uniformity and that we might run into unique bugs that will never happen on the real builders. Happy to embrace the chaos if Miles is.